### PR TITLE
GSOC-2020 Playing with Flex.

### PR DESCRIPTION
**Pull request by sarannng**
Please go throught [How to contribute guide](https://github.com/divyanshub024/awesome-dart-pad/blob/master/Contribution.md) before creating a PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,10 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
 - [Buttons-Flutter-web](https://dartpad.dev/214eee8a820ba4bcff73279a93281a53) by [Vishwesh Soni](https://github.com/vishweshsoni)(31/03/2020)
 
   [<img src = "/art/buttons.gif" width=240>](https://dartpad.dev/214eee8a820ba4bcff73279a93281a53)
+  
+  - [Playing-with-Flex](https://dartpad.dev/2889325c3f7b0ad13a6950ad065f7bbe) by [Sarang Pal](https://github.com/sarannng)(31/03/2020)
```
Comment (mkiisoft) [inline README.md]: remove extra space

**Review by mkiisoft — CHANGES_REQUESTED** [review]

**Comment by mkiisoft** [comment]
Would be nice to have a detailed PR telling what you did, a GIF/Screenshot of your implementation, etc instead of the detail message:

"Please go throught How to contribute guide before creating a PR."

Thanks for your contribution!

File: README.md (hunk 1 above)
Comment (mkiisoft) [reply README.md]: The space is for line 133

**Review by mkiisoft — CHANGES_REQUESTED** [review]

**Review by divyanshub024 — APPROVED** [review]
Thanks a lot for your contribution! 🙏

**Pull request merged**